client/src/app/components/AppDialogShell.vue

```diff
@@ -180,7 +181,7 @@ export default {
 
 .dialog-body {
   overflow-y: auto;
-  max-height: calc( 90vh - 20vh );
+  min-height: 40vh;
```

Comment:
It seems the min-height is added to allow room for the date picker. Yes?  Yet why do we need to remove the max-height rule which limits the dialog to the size of the display and forces scroll bars inside the dialog for content that reaches below the fold.